packages/docs/src/components/styleguide.tsx

```diff
@@ -6,77 +6,59 @@
  */
 
 import * as classNames from "classnames";
+import { IPageData, IPageNode, isPageNode } from "documentalist/dist/client";
```

Comment:
importing the root is not possible on the client cuz it'll include postcss and typescript.